### DISCUSSION TITLE
How to use inbox attribute?

### DISCUSSION THREAD

**Discussion by xsmyqf** [TriliumNext/Trilium#3219] [General]
Hi,I searched documents and examples but can not find any tips：

1. I want to know how to use inbox attribute.
2. how to let a note with attribute "#template #workspace " not show in the "new tab" page.

**Comment by zadam** [comment]
> I want to know how to use inbox attribute.

Mark some note with label `#inbox` and the notes you create with "new note" button in the sidebar will be put there instead of the day notes.

> how to let a note with attribute "#template #workspace " not show in the "new tab" page.

There isn't a way currently. But I'm curious, what's the use case to use both of these labels together?

**Reply by xsmyqf** [reply]
thanks,I wrongly regarded "create child note" as "new note" so that I can not test the "inbox attribute".

> There isn't a way currently. But I'm curious, what's the use case to use both of these labels together?

I want to create a workspace template which other people use to create their workspace,so I want to hide the template in that page.

**Reply by zadam** [reply]
Ok, Trilium isn't really meant to be used by multiple people, but I of course can't tell you what to do and what not.

I added a change to not display "workspace templates" in the new tab, will be in 0.56.1

**Reply by xsmyqf** [reply]
thanks,I have another question to ask:

I tried to use runOnNoteCreation when I first use trilium last week,but I can not trigger it.I googled this and use the solution in [2166](https://github.com/zadam/trilium/issues/2166).

but I really want to know how to trigger runOnNoteCreation.Can you explain it?

**Reply by zadam** [reply]
The idea of `runOnNoteCreation` is that you define it on some subtree (as inheritable) and it gets triggered when a note is created within this subtree. But your question prompted me to test it and it seems it has been broken for a while. I've fixed it. In the mean time, check if `runOnChildNoteCreation` does not suite you as well.

**Reply by xsmyqf** [reply]
runOnChildNoteCreation is okay,you are very efficiency!
